operations/power_management/Power_On_Compute_and_IO_Cabinets.md

```diff
@@ -40,40 +42,47 @@ After the CDU is switched on and healthy, the liquid-cooled PDU circuit breakers
+   This command resumes the `hms-discovery` job which initiates power-on of the liquid-cooled cabinets. The `--stage cabinet-power`
+   option controls power only to liquid-cooled cabinets.
 
    If `sat bootsys` fails to schedule `hms-discovery` with the following message, then delete and recreate the cron job.
 
-   ```
+   ```text
    ERROR: The cronjob hms-discovery in namespace services was not scheduled within expected window after being resumed.
    ```
 
-   If `sat bootsys` fails to power on the cabinets through `hms-discovery`, use CAPMC to manually power on the cabinet chassis, compute blade slots, and all populated switch blade slots \(1, 3, 5, and 7\). This example shows cabinets 1000-1003.
+   If `sat bootsys` fails to power on the cabinets through `hms-discovery`, then use CAPMC to manually power on the cabinet chassis,
+   compute blade slots, and all populated switch blade slots \(1, 3, 5, and 7\). This example shows cabinets 1000-1003.
 
    ```bash
    ncn-m001# cray capmc xname_on create --xnames x[1000-1003]c[0-7] --format json
    ncn-m001# cray capmc xname_on create --xnames x[1000-1003]c[0-7]s[0-7] --format json
    ncn-m001# cray capmc xname_on create --xnames x[1000-1003]c[0-7]r[1,3,5,7] --format json
    ```
 
-### POWER ON STANDARD RACK PDU CIRCUIT BREAKERS
+<a name="power-on-standard"></a>
+
+### Power On Standard Rack PDU Circuit Breakers
 
 1. Switch the standard rack compute and I/O cabinet PDU circuit breakers to ON.
 
-   This applies power to the server BMCs connects them to the management network. Compute and I/O nodes **do not power on and boot automatically**. The Boot Orchestration Service \(BOS\) brings up compute nodes and User Access Nodes \(UANs\).
+   This applies power to the server BMCs and connects them to the management network. Compute and I/O nodes
+   **do not power on and boot automatically**. The Boot Orchestration Service \(BOS\) brings up compute nodes and User Access Nodes \(UANs\).
 
    If necessary, use IPMI commands to power on individual servers as needed.
 
-1. Make sure all system management network switches and Slingshot network switches are powered on in each rack and there are no error LEDS or hardware failures.
+1. Verify that all system management network switches and Slingshot network switches are powered on in each rack, and that
+   there are no error LEDS or hardware failures.
+
+1. Bring up the Slingshot Fabric.
+   Refer to the following documentation for more information on how to bring up the Slingshot Fabric:
 
-1.  Bring up the Slingshot Fabric.
-    Refer to the following documentation for more information on how to bring up the Slingshot Fabric:
-    -  The *Slingshot Administration Guide* PDF for HPE Cray EX systems.
-    -  The *Slingshot Troubleshooting Guide* PDF.
+      * The *Slingshot Administration Guide* PDF for HPE Cray EX systems.
+      * The *Slingshot Troubleshooting Guide* PDF.
```

Comment:
```suggestion
      * The *HPE Slingshot Troubleshooting* PDF.
```

Reply:
This got merged before I could incorporate it, so I've added it to another linting PR of mine:
https://github.com/Cray-HPE/docs-csm/pull/1520